pkg/runtime/runtime.go

```diff
@@ -749,14 +754,7 @@ func (a *DaprRuntime) initInputBinding(c components_v1alpha1.Component) error {
 	}
 
 	log.Infof("successful init for input binding %s (%s)", c.ObjectMeta.Name, c.Spec.Type)
-	if _, ok := a.inputBindings[c.Name]; !ok {
-		go func() {
```

Comment:
Can you please double-check if we do not need go routine to reading from binding ?

Reply:
Yes, we do binding.Read is a blocking method and might block main thread